### DISCUSSION TITLE
Heroku Setup, etc.

### DISCUSSION THREAD

**Discussion by hyperdemon** [titaniumnetwork-dev/Corrosion#6] [Q&A] [answered]
yea im new i have no experience to code so ima ask 
HOW THE HELL DO YOU INSTALL THIS ON HEROKU

in simple dumbass terms

thanks

**Comment by nevr13** [comment]
chill bro

**Comment by hyperdemon** [comment]
yea my bad

**Comment by QuiteAFancyEmerald** [comment]
Please read #1

**Comment by Xproassassinn** [comment]
Well, first u need to go to https://signup.heroku.com/ and make your account once u have done so come back to this project and click the reply to Heroku button it should redirect u to the Heroku creation page, enter a name that it allows and click create app once it finishes it build u can open the app that's all there is too it hope this helps if u need anymore help feel free to ask more questions :)

**Reply by hyperdemon** [reply]
i did, ive deployed 20 different things with heroku, but theres no button to deploy on heorku on the page.

**Comment by QuiteAFancyEmerald** [comment, marked as answer]
Check out https://github.com/BinBashBanana/Corrosion-Heroku
You still clearly did not setup things correctly but yeah :D

**Reply by hyperdemon** [reply]
thanks

**Reply by ghost** [reply]
I had the same problem, thanks!!!

**Comment by nevr13** [comment]
heroku kids are happy now ._.
